static/app/components/events/eventCause.tsx

```diff
@@ -5,7 +5,7 @@ import uniqBy from 'lodash/uniqBy';
 
 import {CommitRowProps} from 'sentry/components/commitRow';
 import {CauseHeader, DataSection} from 'sentry/components/events/styles';
-import {Panel} from 'sentry/components/panels';
+import Panel from 'sentry/components/panels/panel';
```

Comment:
Can we rename the panel export to be a named export instead of making it a default export?

Reply:
what's the functional benefit?
it'll be more work for me to coordinate that across sentry and getsentry so i'd rather not care about it and leave this as they are.

Reply:
It's one less default export and I thought it would be less changes to the PR so I thought it would be a quick win. If it requires coordinating with gs, then we can just do it later on at some point